_posts/2015-06-10-agile-fluency.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "When a company turns agile"
+date: "2015-06-10 07:28:00"
+icon: refresh
+tags: agile scrum company
+categories: scrum
+authors: ["Anja Bärike"]
+---
+
+Did you ever wonder how companies that are going the agile way are developing further over time? And how long it actually takes to get from level to level? Or how many companies achieve certain levels of agility?
+
+Here are all the answers:
+
+{% image blog-agile-fluency.jpg %}{% endimage %}
+
+The picture shows the so called Agile Fluency Model. The numbers are based on questions that had been asked to companies who changed to an agile methodology some years ago.
```

Comment:
"to companies whom" fits better, although I am no english grammar nerd :dancers: